app/components/check_icon.rb

```diff
@@ -0,0 +1,2 @@
+class CheckIcon < ViewComponent::Base
```

Comment:
:+1: We could use this in the `ProviderInterface` too, where we currently have a partial — we can pick it up when we fix up all the account UI next week

Reply:
👍